account_banking_mandate/models/account_banking_mandate.py

```diff
@@ -145,6 +145,14 @@ def create(self, vals=None):
             vals['unique_mandate_reference'] = \
                 self.env['ir.sequence'].next_by_code(
                     'account.banking.mandate') or 'New'
+        if self._context.get('mandate_bank_partner_view'):
+            partner_id = self._context['partner_id']
```

Comment:
Don't use internal variable. Use the same as the second line (`self.env.context`)